RNKeychainManager/RNKeychainManager.m

```diff
@@ -270,7 +270,7 @@ - (OSStatus)deleteCredentialsForServer:(NSString *)server
 {
   NSError *aerr = nil;
   LAContext *context = [LAContext new];
-  BOOL canBeProtected = [context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:&aerr];
+  BOOL canBeProtected = [context canEvaluatePolicy:LAPolicyDeviceOwnerAuthentication error:&aerr];
```

Comment:
This policy requires iOS 9. Since this library is targeting iOS 9, this shouldn't need any OS version checks.